tools/rna_tools/sshmm/sshmm.xml

```diff
@@ -0,0 +1,234 @@
+<tool id="sshmm" name="ssHMM" version="1.0.7">
+    <description>- RNA sequence-structure motif finder</description>
+    <requirements>
+        <requirement type="package" version="1.0.7">sshmm</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    python '$__tool_directory__/fasta_report_sequence_lengths.py' $genome_fasta_file > chrom.sizes &&
```

Comment:
quoting the genome_fasta_file

Reply:
You mean like '${genome_fasta_file}' ? Does this help in case it contains white spaces etc ?

Reply:
Jupp.